todo_app/data/trello_service.py

```diff
@@ -24,16 +25,21 @@ def send_request(self, url, query, action="GET"):
         return requests.request(action, url, headers=self.headers, params=query)
 
     def get_cards_from_board(self):
-        url = f"{self.base_url}1/boards/{self.board_id}/cards"
-        response = self.send_request(url, self.base_query)
+        url = f"{self.base_url}1/boards/{self.board_id}/lists"
+        query = self.base_query | {
+            'cards': "open"
+        }
+        response = self.send_request(url, query)
 
-        return response.json()
+        lists = response.json()
+        items = []
 
-    def get_list_from_board(self, list_id):
-        url = f"{self.base_url}1/lists/{list_id}"
-        response = self.send_request(url, self.base_query)
+        for list in lists:
+            print(list)
```

Comment:
This sort of print is really useful for debugging, but could cause us a headache if it ends up in production since this list may well contain sensitive data (particularly in terms of personal data for our users) - in the cards array it's easy to imagine data like "go see doctor about <x>" or "send letter to <address>" that we might not want everyone with log access to be able to see. Very minor for our project exercise, but worth considering for real applications!